src/main/java/com/hillel/items_exchange/validator/UserRegistrationDtoValidator.java

```diff
@@ -35,7 +39,7 @@ public void validate(Object o, Errors errors) {
             throw new UnprocessableEntityException(EMAIL_DUPLICATE);
```

Comment:
Would you mind extract exceptions messages to property file?

Reply:
Understood)